kv/txn_coord_sender.go

```diff
@@ -758,28 +756,26 @@ func (tc *TxnCoordSender) heartbeat(id string) {
 					Reply: reply,
 				}
 
-				task := tc.stopper.StartTask()
-				if !task.Ok() {
+				if !tc.stopper.RunTask(func() {
```

Comment:
That's fine - my point is not about the presence of `continue` specifically; I'd like to see the return values removed from RunTask and RunAsyncTask before this PR is merged; the `continue` is just a symptom (i think).